apps/judicial-system/api/src/app/app.module.ts

```diff
@@ -14,6 +14,7 @@ import { SharedAuthModule } from '@island.is/judicial-system/auth'
 
 import { environment } from '../environments'
 import { BackendApi } from './data-sources/backend'
+import { EventLogModule } from './modules/event-log/eventLog.module'
```

Comment:
Please consider adding `EventLogModule` to the module barrel file and import from there as is done fo other modules.